test/end-to-end/helpers/mvCaseHelper.js

```diff
@@ -161,6 +161,16 @@ async function redactTextAndThenRemovingRedactionTest(I, caseId, mediaType) {
   await I.redactTextAndThenRemoveRedaction();
 }
 
+async function redactFirstPageTest(I, caseId, mediaType) {
+  await executeTestsOnPreview(I, caseId, mediaType);
```

Comment:
We have an existing test script that redacting content on "First Page"  using below functions:
1) Draw Box function
2) Redact Text Function

So exclude this text  from master pipeline  execution and tag to "nightly build(np)"

![image](https://user-images.githubusercontent.com/87771652/212658265-8eff7643-dfd1-4c88-8628-e9aa60ce70dd.png)

Reply:
I had a call with Michael and the new feature tests will run both AAT as well as on Preview